### PR TITLE
[UPD] copier template

### PR DESCRIPTION
**Pull request by sergiocorato**

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM

File: .github/workflows/pre-commit.yml [hunk 1]
```diff
@@ -13,17 +13,10 @@ jobs:
...
     runs-on: ubuntu-22.04
     steps:
-<<<<<<< before updating
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v4
-        with:
-          python-version: "3.10"
-=======
       - uses: actions/checkout@v3
       - uses: actions/setup-python@v2
```
Comment (pedrobaeza) [inline .github/workflows/pre-commit.yml:17]: Shouldn't this be v4?

Comment (sergiocorato) [reply .github/workflows/pre-commit.yml:17]: I don't know, I used `copier update --UNSAFE`

Comment (sergiocorato) [reply .github/workflows/pre-commit.yml:17]: v2 seems correct, looking for other repos

**Review by grindtildeath — APPROVED** [review]

**Comment by sbidoul** [comment]
Ow, thanks for the update @sergiocorato. I'll try to keep in mind to double check why https://github.com/OCA/maintainer-tools/blob/master/tools/copier_update.py did push changes with merge conflicts.

**Pull request merged**